src/Microsoft.DotNet.Wpf/test/Directory.Build.props

```diff
@@ -1,20 +1,56 @@
 <Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
   <PropertyGroup>
-    <WpfTest>true</WpfTest>
-  </PropertyGroup>
-  <Import Condition="Exists('../Directory.Build.props')" Project="../Directory.Build.props" />
-  <PropertyGroup>
-    <WpfTestsDir>$(MsBuildThisFileDirectory)</WpfTestsDir>
-    <SignAssembly>false</SignAssembly>
-    <IsTestProject>true</IsTestProject>
-    <IsShipping>false</IsShipping>
-    <EnableXlfLocalization>false</EnableXlfLocalization>
-    <CLSCompliant>false</CLSCompliant>
+    <WpfTestBasePath>$(MsBuildThisFileDirectory)</WpfTestBasePath>
+    <DrtCommonRoot>$(WpfTestBasePath)\Common\DRT</DrtCommonRoot>
+    <WpfSharedRoot>$(WpfTestBasePath)\Shared</WpfSharedRoot>
+    <PublishRoot>$(RepoRoot)artifacts\test\</PublishRoot>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
This like `CopyLocalLockfileAssemblies` should be configured centrally for test projects (or for wpfTest==true) and not per-project/ad-hoc.